build.gradle

```diff
@@ -39,10 +39,6 @@ ext {
 
 
 ext {
-    // libs
-    wordPressFluxCVersion = '1.36.0'
-    wordPressUtilsVersion = '2.3.0'
```

Comment:
Praise (❤️): This is awesome, with one stone you got two birds, `FluxC` and `Utils`, bullseye, this project is now lips dependency free!!! 🎉